src/pages/DailyGardening/index.tsx

```diff
@@ -73,17 +73,32 @@ export const DailyGardening = () => {
 
   useEffect(() => {
     if (gardenId) {
-      getGardenData(gardenId);
+      const dateISO: string = formatISO(new Date(), {
+        representation: "date",
+      });
+      getGardenData(gardenId, dateISO);
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [gardenId]);
 
   const completeTaskHandler = useCallback(
     async (rule: Rule) => {
+      const localeDate: Date = new Date();
+      const utcDate: Date = new Date(
+        Date.UTC(
```

Comment:
I think here no need to explicitly add the type, typescript would infer from value
![image](https://user-images.githubusercontent.com/22023256/132505866-76e3b94a-0f20-43ce-9e36-e5e42d1e24ab.png)

Reply:
Removed the type!